File: CS/GridControlDetailView/CustomGrid/CustomHandler.cs
using System;
using System.Collections.Generic;
using System.Linq;
using DevExpress.XtraGrid.Views.Base.Handler;

namespace CustomGrid
{
    class CustomHandler : BaseViewHandler
    {
        public CustomHandler(CustomView view)
            : base(view)
        {
        }
    }
}

File: VB/GridControlDetailView/CustomGrid/CustomHandler.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports DevExpress.XtraGrid.Views.Base.Handler

Namespace CustomGrid
	Friend Class CustomHandler
		Inherits BaseViewHandler

		Public Sub New(ByVal view As CustomView)
			MyBase.New(view)
		End Sub
	End Class
End Namespace

